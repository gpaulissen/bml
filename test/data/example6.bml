#CUT nt_module
2C  Stayman
  2D No major
  2N 4-4 majors, minimum
  3C 4-4 majors, maximum
2red Transfer
2S  Minor suit stayman
2N  Invitational
#ENDCUT

#VUL N0

#SEAT 12

#HIDE
1N 12--14
  #PASTE nt_module

  So when we're not vulnerable we open 1NT 12--14 in 1st and 2nd seat.
  
#SEAT 34
  
#HIDE
1N 14--16
  #PASTE nt_module

  But in third and fourth seat it is 14--16.
  
#VUL Y0
  
#SEAT 0
  
#HIDE
1N 14--16
  #PASTE nt_module

  When we're vulnerable we always open 1NT 14--16.

1N-
#PASTE nt_module
