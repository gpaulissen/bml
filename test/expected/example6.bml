#VUL Y0

#SEAT 0

1N = 14--16
  2C = Stayman
    2D = No major
    2N = 4-4 majors, minimum
    3C = 4-4 majors, maximum
  2D = Transfer
  2H = Transfer
  2S = Minor suit stayman
  2N = Invitational

#VUL N0

#SEAT 12

1N = 12--14
  2C = Stayman
    2D = No major
    2N = 4-4 majors, minimum
    3C = 4-4 majors, maximum
  2D = Transfer
  2H = Transfer
  2S = Minor suit stayman
  2N = Invitational

#VUL N0

#SEAT 34

1N = 14--16
  2C = Stayman
    2D = No major
    2N = 4-4 majors, minimum
    3C = 4-4 majors, maximum
  2D = Transfer
  2H = Transfer
  2S = Minor suit stayman
  2N = Invitational
