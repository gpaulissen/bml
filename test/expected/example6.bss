*00{}=NYYYYYY
551N=NYYYYYY012--14
551NP2C=NYYYYYY008Stayman
551NP2CP2D=NYYYYYY008No major
551NP2CP2N=NYYYYYY04-4 majors, minimum
551NP2CP3C=NYYYYYY0084-4 majors, maximum
551NP2S=NYYYYYY008Minor suit stayman
551NP2N=NYYYYYY0Invitational
551NP2D=NYYYYYY008Transfer
551NP2H=NYYYYYY008Transfer
651N=NYYYYYY014--16
651NP2C=NYYYYYY008Stayman
651NP2CP2D=NYYYYYY008No major
651NP2CP2N=NYYYYYY04-4 majors, minimum
651NP2CP3C=NYYYYYY0084-4 majors, maximum
651NP2S=NYYYYYY008Minor suit stayman
651NP2N=NYYYYYY0Invitational
651NP2D=NYYYYYY008Transfer
651NP2H=NYYYYYY008Transfer
061N=NYYYYYY014--16
061NP2C=NYYYYYY008Stayman
061NP2CP2D=NYYYYYY008No major
061NP2CP2N=NYYYYYY04-4 majors, minimum
061NP2CP3C=NYYYYYY0084-4 majors, maximum
061NP2S=NYYYYYY008Minor suit stayman
061NP2N=NYYYYYY0Invitational
061NP2D=NYYYYYY008Transfer
061NP2H=NYYYYYY008Transfer
