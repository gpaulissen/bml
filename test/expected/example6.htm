<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Strict//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd'>
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><title>No title supplied</title><style type="text/css">div.start {
    display: inline-block;
    vertical-align: top;
    width: 60px;
}
    
div.bidtable {
    width:700px;
    padding-bottom: 20px;
}    

div.bidtable ul {
    list-style-type: none;
    margin: 0;
    padding: 0;
}

div.bidtable li ul
{
    list-style-type: none;
    margin: 0;
    padding: 0;
    margin-left: 60px;
}

div.tree {
    list-style-type: none;
    padding-bottom: 20px;
}

div.tree ul {
    list-style-type: none;
    /* vline.png */
    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAAKAQMAAABPHKYJAAAAA1BMVEWIiIhYZW6zAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH1ggGExMZBky19AAAAAtJREFUCNdjYMAEAAAUAAHlhrBKAAAAAElFTkSuQmCC) repeat-y;
    margin: 0;
    padding: 0;
    margin-left: 10px;
}

div.tree li.node {
    margin: 0;
    padding: 0 12px;
    line-height: 20px;
    /* node.png */
    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAUAQMAAACK1e4oAAAABlBMVEUAAwCIiIgd2JB2AAAAAXRSTlMAQObYZgAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9YIBhQIJYVaFGwAAAARSURBVAjXY2hgQIf/GTDFGgDSkwqATqpCHAAAAABJRU5ErkJggg==) no-repeat;
}

div.tree td.node {
    vertical-align: top;
    width: 60px;
}

div.tree li.last {
    /* lastnode.png */
    background: #fff url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAUAQMAAACK1e4oAAAABlBMVEUAAwCIiIgd2JB2AAAAAXRSTlMAQObYZgAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9YIBhQIIhs+gc8AAAAQSURBVAjXY2hgQIf/GbAAAKCTBYBUjWvCAAAAAElFTkSuQmCC) no-repeat;
}

div.tree li.root {
    background: #fff none no-repeat;
}

.ccolor{color:Green}
.dcolor{color:Orange}
.hcolor{color:Red}
.scolor{color:Black}
</style></head><body><p>So when we're not vulnerable we open 1NT 12&ndash;14 in 1st and 2nd seat.</p><p>But in third and fourth seat it is 14&ndash;16.</p><p>When we're vulnerable we always open 1NT 14&ndash;16.</p><div class="bidtable"><ul><li><div class="start">1NT</div><ul><li><div class="start">2<span class="ccolor">&clubs;</span></div>Stayman<ul><li><div class="start">2<span class="dcolor">&diams;</span></div>No major</li><li><div class="start">2NT</div>4-4 majors, minimum</li><li><div class="start">3<span class="ccolor">&clubs;</span></div>4-4 majors, maximum</li></ul></li><li><div class="start">2red</div>Transfer</li><li><div class="start">2<span class="scolor">&spades;</span></div>Minor suit stayman</li><li><div class="start">2NT</div>Invitational</li></ul></li></ul></div></body></html>